config/m4/compiler.m4

```diff
@@ -165,6 +166,60 @@ AS_IF([test "x$with_avx" != xyes],
       ])
 
 
+AC_ARG_ENABLE(aarch64-tuned,
```

Comment:
maybe worth putting it in a separate .m4 file?

Reply:
I do not have strong preferences here. Something like aarch64.m4 ?